Lib/traceback.py

```diff
@@ -622,15 +630,32 @@ def format(self, *, chain=True):
         The message indicating which exception occurred is always the last
         string in the output.
         """
-        if chain:
-            if self.__cause__ is not None:
-                yield from self.__cause__.format(chain=chain)
-                yield _cause_message
-            elif (self.__context__ is not None and
-                not self.__suppress_context__):
-                yield from self.__context__.format(chain=chain)
-                yield _context_message
-        if self.stack:
-            yield 'Traceback (most recent call last):\n'
-            yield from self.stack.format()
-        yield from self.format_exception_only()
+
+        stack = []
```

Comment:
It's a bit confusing that we have a local variable `stack` now but also an attribute `exc.stack` (completely unrelated).